manifest-rhel-9.2.yaml

```diff
@@ -0,0 +1,157 @@
+# Manifest for RHCOS based on RHEL 9.2 (C9S for now)
+
+rojig:
+  license: MIT
+  name: rhcos
+  summary: OpenShift 4
+
+variables:
+  osversion: "rhel-9.2"
+
+# Include manifests common to all RHEL and CentOS Stream versions and manifest
+# common to RHEL 9 & C9S variants
+include:
+  - common.yaml
+  - common-el9.yaml
+
+# Starting from here, everything should be specific to RHCOS based on RHEL 9.2
+
+# See "Notes about repositories" in `docs/development.md`
+repos:
+  - baseos
+  - appstream
+  # For NFV & Virtualization SIG GPG keys
+  - extras-common
+  - sig-nfv
+  # Include RHCOS 9.0 repo for oc, hyperkube, cri-o, conmon-rs
+  - rhel-9.0-server-ose-4.13
+  # Uncomment and switch to the following once ready
+  # - rhel-9.2-baseos
+  # - rhel-9.2-appstream
+  # - rhel-9.2-fast-datapath
+  # - rhel-9.2-server-ose-4.13
+
+arch-include:
+  s390x:
+    # Temporary repo for openvswitch on s390x
```

Comment:
I think only shipping a particular package on a particular architecture is going to trip up ART's consistentency checks; we should use the same content across all arches.